packages/teamviewer-deb/teamviewer-deb.pacscript

```diff
@@ -0,0 +1,9 @@
+name="teamviewer-deb"
+gives="teamviewer"
+breaks="${gives}-bin"
+version="15.33.7"
+description="TeamViewer provides easy, fast and secure remote access and meeting solutions"
+hash="29b9a5e7b0673a5159c1ec492d0753c1fabfb0b1bb961e8445a68f789cbb87db"
+maintainer="Obsidian <emil@aarestrup.nu>"
+url="https://download.teamviewer.com/download/linux/teamviewer_amd64.deb"
```

Comment:
Does TeamViewer not have a versioned URL?

Reply:
They do it's
`https://download.teamviewer.com/download/linux/version_(version start number ie: 13 or 14 in case of latest it would be 15)x/teamviewer_(full version tag ie: 15.33.7)_amd64.deb`

Reply:
@ObsidianPresidium